src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -60,24 +60,24 @@
 ;;; keep these overrides.
 (def tls-1-dot-3-ciphers
   ["TLS_AES_128_GCM_SHA256"
-   "TLS_AES_256_GCM_SHA384"
-   "TLS_CHACHA20_POLY1305_SHA256"])
+   "TLS_AES_256_GCM_SHA384"])
 
 (def tls-1-dot-2-ciphers
-  ["TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
-   "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"
+  ["TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"
    "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384"
-   "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"
-   "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256"
-   "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"])
+   "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"
+   "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
```

Comment:
So I think this is a better ordering of the suites, but I'm going to pull @mwaggett in on this because she did the leg work on why they are in the order they are now (see https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/pull/231)

Reply:
The site doesn't really have the correct ordering.  I double checked with Jeremy on what the ordering should be and https://github.com/puppetlabs/puppet-enterprise-modules/blob/main/modules/puppet_enterprise/functions/ciphers.pp is the right order.  Generally, we want 256 before 128, and certain new ciphers before other older ones.

Reply:
ahkay, well in that case, could you or @LivingInSyn update https://confluence.puppetlabs.com/display/SRE/TLS+Usage+Standard to match (and remove references to Mozilla's ordering if that's not accurate/what we're using)?

Reply:
Just updated, waiting on final check by @LivingInSyn to make sure I didn't say anything incorrect.

Reply:
Looks like the Mozilla references are still in there. I can remove them and @LivingInSyn can check that too. 😁

Reply:
It's still based on the Mozilla Intermediate list.  Just, for some reason, the Mozilla wiki lists them in a weird order with 128s before 256s.

Reply:
I'm also not 100% sure if order actually matters or is followed by the various services, and I haven't looked into a way to test that.

Reply:
![Jake Peralta from Brooklyn Nine-Nine saying 'cool' a lot](https://media.giphy.com/media/l41lNp8dHpfaJE25O/giphy.gif)